### ISSUE TITLE
App router interceptors and default language middleware 

### ISSUE THREAD

**Issue by RexGalicie** [i18nexus/next-i18n-router#37]
Basically all information is here 
https://github.com/vercel/next.js/issues/58857

**Comment by i18nexus** [comment]
Can you clarify whether this is an issue with this library or with next? It appears that you were able to reproduce the issue even without this library: https://github.com/vercel/next.js/issues/58857

**Comment by RexGalicie** [comment]
Yeah true, I do believe it is more related to Nextjs  as it faced with or without [next-i18n-router](https://github.com/i18nexus/next-i18n-router)

**Comment by i18nexus** [comment]
According to this issue (https://github.com/vercel/next.js/issues/49662) many people have been experiencing problems with intercepted routes. After testing your repo, it seems like a bug within Next. Perhaps following that issue and related issues will help. I'm going to have to close this issue since there are no signs that this is caused by a bug with this library. Hope the Next.js team can figure it out!
